fs

```diff
@@ -24,10 +24,11 @@ def cli(ctx, fleetctl_endpoint, timeout):
 @click.argument('service-name', type=str)
 @click.argument('unit-file', type=click.Path(exists=True))
 @click.option('--count', type=int, default=3)
+@click.option('--restart', type=click.Choice(['always', 'no']), default=None, help='Restart strategy for unit. If not specified, it uses what is defined in the unit.')
```

Comment:
Don't you want to add all restart options? (no, on-success, on-failure, on-abnormal, on-watchdog, on-abort, or always)

Also can you change the wording of the help message to "Restart strategy for unit. If not specified, the restart strategy as defined in the unit will be used". I don't really like "it" in these sentences.
Or maybe shorter "Override the unit's restart strategy"?